src/main/java/org/folio/rest/impl/util/DataImportKafkaTopic.java

```diff
@@ -0,0 +1,27 @@
+package org.folio.rest.impl.util;
+
+import org.folio.kafka.services.KafkaTopic;
+
+public enum DataImportKafkaTopic implements KafkaTopic {
+  DI_COMPLETED("DI_COMPLETED"),
```

Comment:
mod-data-import does not publish in this topics neither. Please review the publisher, let's leave only those topics that mod-data-import needs to publish to